frontend/src/scenes/billing/BillingProduct.tsx

```diff
@@ -277,6 +281,23 @@ export const BillingProduct = ({ product }: { product: BillingProductV2Type }):
                                                 </div>
                                             </Tooltip>
                                         </div>
+                                    ) : isTemporaryFreeProduct ? (
```

Comment:
Nit: do you need `isTemporaryFreeProduct`. It seems to be an unneeded complication. Given this will be removed on August 8th and it's specific to the data warehouse, maybe just use that as the check? 

It's not really needed until we support the content of this box to be dynamic from billing.

Reply:
I do actually want to keep it there and not make it specific to DW. I think the way we are doing this launch is something we will prob do in the future with other launches - roll out pricing for everyone, give beta users 30 days notice, during which they will need to see their usage. The notice on the interior _is_ specific to data warehouse launch, but it'll be specific to every product launch, so I don't really know how to make it more reusable right now. And I don't really want to add a bunch of fields to billing data to fill it out.